Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -1242,7 +1242,9 @@ -(UIEdgeInsets)safeAreaInsets
 {
     UIEdgeInsets insets = UIEdgeInsetsMake(0.0, 0.0, 0.0, 0.0);
     if (@available(iOS 11.0, *)) {
-        insets =  [UIApplication sharedApplication].keyWindow.safeAreaInsets;
+        insets = [UIApplication sharedApplication].keyWindow.safeAreaInsets;
+    } else {
+        insets.top = [[UIApplication sharedApplication] isStatusBarHidden] ? 0 : 20.0;
```

Comment:
i thought the minimum insets.top we needed was "20" , especially if the statusBar was hidden.
Here, it looks like if statusbar is hidden, the top is 0?

Something along this lines:
https://stackoverflow.com/questions/46880614/safe-area-insets-change-when-hiding-status-bar-ios-11